Makefile

```diff
@@ -28,8 +28,10 @@ uninstall:
 deploy:
 	if [[ -z "$(NEW_VERSION)" ]]; then echo "Please add a value to the NEW_VERSION variable"; exit 1; fi
 	`Scripts/update_makefile.sh`
+	`Scripts/dev_deps_hide.rb`
 	`Scripts/update_danger_version.sh`
 	`Scripts/update_changelog.sh`
 	make docs
 	`Scripts/commit_new_version.sh`
 	`Scripts/create_homebrew_tap.sh`
+	`Scripts/dev_deps_unhide.rb`
```

Comment:
I added these two extra steps to the deploy, ruby because everyone has ruby installed by default